app/validators/contribution-date.js

```diff
@@ -0,0 +1,12 @@
+import validateDateRange from 'opensource-challenge-client/validators/date-range'
+
+import moment from 'moment'
+
+export default function validateContributionDate() {
+  return (key, newValue, oldValue, changes, model) => {
+    return validateDateRange({
+      min: model.get('challenge.startsOn'),
```

Comment:
I think this doesn't work yet as mirage does not create the contribution model correctly.